acquia_lift.admin.wizard.inc

```diff
@@ -16,7 +16,10 @@
  */
 function acquia_lift_personalize_campaign_wizard_process_bar_alter(&$form, &$form_state, $form_id) {
   $agent = $form['#agent'];
-
+  $nested = acquia_lift_get_nested_tests($agent);
```

Comment:
Yeah, I actually think maybe the sync with Lift button can just go away now that syncing happens at very specific times using batch api. As to reset data, that should probably only show up on the report for the test?